application/views/scripts/user/show.phtml

```diff
@@ -40,6 +40,32 @@ use Icinga\Data\Selectable;
             <th><?= $this->translate('Last modified'); ?></th>
             <td><?= $user->last_modified === null ? '-' : $this->formatDateTime($user->last_modified); ?></td>
         </tr>
+        <tr>
+            <th><?= $this->translate('Role Memberships'); ?></th>
+            <td>
+            <?php $roleObj =  $userObj->getRoles(); ?>
+            <?php if (! empty($roleObj)): ?>
+                <ul class="role-memberships">
+                <?php foreach($roleObj as $role): ?>
+                    <li>
+                    <?php if ($this->allowedToEditRoles): ?>
+                        <?= $this->qlink(
+                            $role->getName(),
+                            'role/edit',
+                            ['role' => $role->getName()],
+                            ['title' => sprintf($this->translate('Edit role %s'), $role->getName())]
+                        ) ?>
+                    <?php else: ?>
+                        <?= $role->getName() ?>
+                    <?php endif ?>
+                    </li>
+                <?php endforeach ?>
```

Comment:
Roles allow spaces in their names. Currently there is no separator between multiple roles, without this it's difficult to differentiate the different roles. (Especially if there's one with spaces in its name) Please add a comma as separator between role names.